lib/ReactViews/Map/BottomLeftBar/BottomLeftBar.tsx

```diff
@@ -53,8 +82,32 @@ const BottomLeftBar: FC = () => {
           </MapIconButton>
         </Box>
       ) : null}
+      {/* Google Logo. Needed for Google Photorealistic 3D Tiles
+       */}
+      {isUsingGooglePhotorealistic3dTiles && (
+        <img
+          height="18px"
+          style={{ paddingLeft: "8px" }}
+          src="build/TerriaJS/images/google_on_non_white_hdpi.png"
+        ></img>
+      )}
```

Comment:
I wonder if it would be better to make this a generic thing.
We could add a `imageOnScreen` trait to `AttributionTraits`, and then show all images of items enabled in the workbench.

This means we can get rid of the hard-coded `"https://tile.googleapis.com/v1/3dtiles/root.json"` in `isUsingGooglePhotorealistic3dTiles` and the Google logo in this repo

Reply:
Do you mean to add a trait that points to an attribution image?

Reply:
And then we upload that to a bucket somewhere and put the url in the catalog item?

Reply:
I actually think it might be better to hard code this and leave it as is - if we make it a trait, then there is no way to enforce people to add attribution. This way there is no choice and we are sure to be in compliance with Googles policy